src/libs/config.js

```diff
@@ -12,3 +12,22 @@ export const getConfig = () => {
  * Flags for hidden features
  */
 export const isCromwellAppVisible = () => getConfig().isCromwellAppVisible
+export const isAxeEnabled = () => {
+  const storedValue = getConfig().isAxeEnabled
+  const isDev = process.env.NODE_ENV === 'development'
+  // It would be nice to be able to enable this on PR sites (production) if the feature flag is enabled,
+  // but unfortunately axe-core only works on page refreshes in that environment.
+  if (!isDev && !!storedValue) {
+    console.log('axe accessibility checking can only be enabled when terra-ui is running in a development environment') // eslint-disable-line no-console
+  }
+  if (isDev) {
+    if (_.isUndefined(storedValue) || storedValue) {
+      console.log('axe accessibility checking is running, and this can negatively impact UI performance') // eslint-disable-line no-console
```

Comment:
```suggestion
      console.log('Axe accessibility checking is running, and this can negatively impact UI performance.') // eslint-disable-line no-console
```

Reply:
They actually start it with a lowercase "a". That's one of the 2 reasons why I chose not to use complete sentences, which I normally am a fan of.